packages/image_picker/image_picker_windows/lib/image_picker_windows.dart

```diff
@@ -112,41 +98,68 @@ class ImagePickerWindows extends ImagePickerPlatform {
     int? imageQuality,
     CameraDevice preferredCameraDevice = CameraDevice.rear,
   }) async {
-    if (source != ImageSource.gallery) {
-      // TODO(azchohfi): Support ImageSource.camera.
-      //                 See https://github.com/flutter/flutter/issues/102115
-      throw UnimplementedError(
-          'ImageSource.gallery is currently the only supported source on Windows');
+    return getImageFromSource(
+        source: source,
+        options: ImagePickerOptions(
+            maxWidth: maxWidth,
+            maxHeight: maxHeight,
+            imageQuality: imageQuality,
+            preferredCameraDevice: preferredCameraDevice));
+  }
+
+  // [ImagePickerOptions] options are not currently supported on macOS. If any
```

Comment:
this do be windows

Reply:
I removed the platforms from all of these comments in all the files; we don't need to say the platform in a platform-specific implementation file in the first place.